src/main/java/org/epam/core/api/RestClientConfig.java

```diff
@@ -0,0 +1,22 @@
+package org.epam.core.api;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.specification.RequestSpecification;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Configuration;
+
+@Slf4j
```

Comment:
Slf4j annotation is present, but there are no logs

Reply:
Thanks for comment! Fixed.